libr/core/disasm.c

```diff
@@ -4209,9 +4209,12 @@ static void ds_print_relocs(RDisasmState *ds) {
 			if (demangle) {
 				demname = r_bin_demangle (core->bin->cur, lang, rel->symbol->name, rel->symbol->vaddr, keep_lib);
 			}
-			r_cons_printf ("; RELOC %d %s @ 0x%08" PFMT64x " + 0x%" PFMT64x,
+			r_cons_printf ("; RELOC %d %s @ 0x%08" PFMT64x,
 					rel->type, demname ? demname : rel->symbol->name,
-					rel->symbol->vaddr, rel->addend);
+					rel->symbol->vaddr);
+			if (rel->addend) {
```

Comment:
addend is signed, can you handle + and - cases here, so the negative values won't look like `+ -0x803`

Reply:
Actually it will look like `+ 0xfffffffffffffffb`.